### ISSUE TITLE
[Epic: js-waku] Public dogfooding RLNaaS web app

### ISSUE THREAD

**Issue by chair28980** [waku-org/js-waku#2164]
## User story

As a Waku CC I want to have a project to share a link to or show on mobile / desktop to people attending Devcon. 
As a Devcon attendee I want to be able to seamlessly use an app shared with me.

**Comment by danisharora099** [comment]
What is this issue about? @chair28980 
cc @weboko

**Comment by weboko** [comment]
@danisharora099 this issue for public dogfooding app that we discussed https://discord.com/channels/1110799176264056863/1282875097480167444/1282875099476791442

we will start working on it right after finishing https://github.com/waku-org/js-waku/issues/2139 and https://github.com/waku-org/js-waku/issues/2158

I will update description with our plan that we had derived at our kickoff meeting. 

Do you want it being assigned to you?

**Comment by danisharora099** [comment]
> Do you want it being assigned to you?

Yeah, sounds good. I'll write a draft proposal for this issue/webapp that we are planning.

**Comment by weboko** [comment]
@danisharora099 let me update description first

**Comment by fryorcraken** [comment]
AFAIK, this is closed to completion, need to stabilize filter further?
Regarding store, we can hardcore sandbox store nodes for now, we are planning work for decentralized store in H1.

Is that the 2 remaining items to close this deliverable? @weboko

**Comment by weboko** [comment]
Resolving as dogfooding app was delivered. Leftover tasks are minor and are not directly influencing RLNaaS.
